clang/unittests/Format/FormatTest.cpp

```diff
@@ -17762,14 +17762,19 @@ TEST_F(FormatTest, ConfigurableSpaceBeforeParens) {
 }
 
 TEST_F(FormatTest, SpaceAfterLogicalNot) {
-  FormatStyle Spaces = getLLVMStyle();
-  Spaces.SpaceAfterLogicalNot = true;
+  auto Spaces = getLLVMStyle();
+  EXPECT_EQ(Spaces.SpaceAfterLogicalNot, FormatStyle::SAN_Never);
 
+  Spaces.SpaceAfterLogicalNot = FormatStyle::SAN_Exclaim;
   verifyFormat("bool x = ! y", Spaces);
   verifyFormat("if (! isFailure())", Spaces);
   verifyFormat("if (! (a && b))", Spaces);
   verifyFormat("\"Error!\"", Spaces);
   verifyFormat("! ! x", Spaces);
```

Comment:
Add a ``not`` test here?

Reply:
`SAN_Exclaim` deprecates `true` and has no effect on `not`, which should be already covered by test cases elsewhere.